daffodil-io/src/main/scala/org/apache/daffodil/io/DataOutputStream.scala

```diff
@@ -24,6 +24,7 @@ import org.apache.daffodil.util.Logging
 import org.apache.daffodil.util.Maybe
 
 sealed abstract class ZeroLengthStatus
+extends Serializable
```

Comment:
Any idea why this needs to be serializable now? Not a big deal, but this feels like a runtime thing that shouldn't need to be serialized?

Reply:
Not sure, I ran into this issue yesterday and thought I had done something completely wrong and redid my changes in a cleaner way and still hit this problem.  I guess its because we are serializing a ZeroLengthStatus as part of the SuspendableOperation now.  Not sure why it wasn't doing it before though

Reply:
But isn't the SuspendableOperation only allocated at Runtime? So its shouldn't need to be serialized. Again, not a big deal, but I feel like maybe I'm missing something.

Reply:
I think this was made serializable before Josh changed the objects to be runtime allocated. So possibly this doesn't *have to* be serializable now.

Reply:
Mike is correct, it no longer needs it.